### PR TITLE
Update index.html to reference new style sheet hack

### PR DESCRIPTION
**Pull request by tti0**

### CODE REVIEW

**Comment by palkerecsenyi** [comment]
I think neutralise may have been spelt incorrectly.

**Comment by tti0** [comment]
British spelling vs American.

**Comment by tti0** [comment]
Sorry, I see my mistake, resolving now.

File: index.html [hunk 1]
```diff
@@ -40,6 +40,7 @@
 		<!-- aesthetic CSS libraries -->
 		<link rel="stylesheet" src="library/animate/animate.css"/>
 		<link type="text/css" rel="stylesheet" href="library/materialize/css/materialize.css"  media="screen,projection"/>
+		<link type="text/css" rel="stylesheet" href="library/netralise.css"/> <!-- our CSS tweak/hack -->
```
Comment (palkerecsenyi) [inline index.html]: neutralise mispelt - instead of netralise please write neutralise

**Pull request merged**